gallery/src/styles/Nivo/NivoLineExample.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { ResponsiveLine } from '@nivo/line';
+import { NIVO_COLORS } from '@sonatype/react-shared-components';
+import { lineData } from './data';
+
+const MyResponsiveLine = () => (
+  <ResponsiveLine data={lineData}
+                  margin={{ top: 50, right: 110, bottom: 50, left: 60 }}
+                  xScale={{ type: 'point' }}
+                  yScale={{
+                    type: 'linear',
+                    stacked: false,
+                    reverse: false
+                  }}
+                  pointColor={{ theme: 'background' }}
+                  pointBorderWidth={2}
+                  pointBorderColor={{ from: 'serieColor' }}
+                  colors={NIVO_COLORS}
+                  axisBottom={{
+                    legend: 'transportation',
+                    legendOffset: 40,
+                    legendPosition: 'middle'
+                  }}
+                  axisLeft={{
+                    legend: 'count',
+                    legendOffset: -40,
+                    legendPosition: 'middle'
+                  }}
+                  axisTop={{
+                    legend: 'Transportation in Different Countries',
+                    legendOffset: -40,
+                    legendPosition: 'middle'
+                  }}
+                  legends={[
+                    {
+                      anchor: 'bottom-right',
+                      direction: 'column',
+                      translateX: 100,
```

Comment:
Is this much custom positioning supposed to be necessary?

Reply:
> Is this much custom positioning supposed to be necessary?

If `translateX` is not there the legend overlaps with the graph, but the symbol stuff is not necessary: 54785e1f8c905b7b6b9a9df87d00569790196681

Reply:
> If translateX is not there the legend overlaps with the graph, but the symbol stuff is not necessary: https://github.com/sonatype/sonatype-react-shared-components/commit/54785e1f8c905b7b6b9a9df87d00569790196681

What I meant is, are we _supposed_ to have to add `translateX` (a fairly low-level property) to get basic legend positioning working or is there something else that we're missing or doing wrong that would be the more correct way to position the legend?

Reply:
> > If translateX is not there the legend overlaps with the graph, but the symbol stuff is not necessary: [54785e1](https://github.com/sonatype/sonatype-react-shared-components/commit/54785e1f8c905b7b6b9a9df87d00569790196681)
> 
> What I meant is, are we _supposed_ to have to add `translateX` (a fairly low-level property) to get basic legend positioning working or is there something else that we're missing or doing wrong that would be the more correct way to position the legend?

As far as I can tell from the documentation, `translateX` and `translateY` are what is used to move it out of the graph after using `anchor` to choose a side to have it on.